packages/node_modules/@ciscospark/internal-plugin-support/test/unit/spec/support.js

```diff
@@ -34,4 +34,11 @@ describe(`plugin-support`, function() {
     });
   });
 
+  describe(`#submitLogs()`, () => {
```

Comment:
yes there is but it is disabled. The comment says that rackspace is broken. Do you want me to enable that and remove the unit test?
![screen shot 2017-06-21 at 4 33 15 pm](https://user-images.githubusercontent.com/22205801/27410923-84b67eb2-569f-11e7-8ad7-5aaeb26bc6c1.png)